src/storage/page_buffer.c

```diff
@@ -14741,6 +14741,33 @@ pgbuf_fix_if_not_deallocated_with_caller (THREAD_ENTRY * thread_p, const VPID *
   return error_code;
 }
 
+int
+pgbuf_fix_if_not_deallocated_with_repl_desync_check (THREAD_ENTRY * thread_p, const VPID * vpid,
+						     PGBUF_LATCH_MODE latch_mode, PGBUF_LATCH_CONDITION latch_condition,
+						     PAGE_PTR * page)
+{
+  int error_code =
+    pgbuf_fix_if_not_deallocated_with_caller (thread_p, vpid, latch_mode, latch_condition, page, ARG_FILE_LINE);
```

Comment:
Nit : Should use pgbuf_fix_if_not_deallocated which is defined as pgbuf_fix_if_not_deallocated_with_caller only for debug code.
Does your code compiles in release ?

Reply:
Changed it now